packages/playwright/src/common/test.ts

```diff
@@ -259,6 +259,7 @@ export class TestCase extends Base implements reporterTypes.TestCase {
 
   _testType: TestTypeImpl;
   id = '';
+  _idWithoutRepeatEach: string | undefined;
```

Comment:
I don't really like this, let's hold off for a bit and collect upvotes? Perhaps the usecase is not the best defined one.